standalone-metastore/metastore-common/src/main/thrift/hive_metastore.thrift

```diff
@@ -2769,6 +2802,11 @@ PartitionsResponse get_partitions_req(1:PartitionsRequest req)
   GetPartitionsByNamesResult get_partitions_by_names_req(1:GetPartitionsByNamesRequest req)
                         throws(1:MetaException o1, 2:NoSuchObjectException o2)
 
+    // retrieve properties
+    PropertyGetResponse get_properties(1:PropertyGetRequest req);
```

Comment:
Should we declare the throwing exception of these two methods? I'm afraid the Thrift would throw transport exception instead if things go wrong.

Reply:
I too agree with @dengzhhu653

Reply:
Agreed.

Reply:
Adding exceptions triggered when creating the manager fail.